src/Components/Publishing/Display/__test__/DisplayPanel.test.js

```diff
@@ -82,20 +82,18 @@ describe("units", () => {
       }
     }
 
-    // TODO: this never gets called because it gets fired before the test runs
-    // it("tracks one time on mount", () => {
-    //   const wrapper = getWrapper()
-    //   const spy = jest.spyOn(wrapper.props().tracking, "trackEvent")
-    //   const instance = wrapper.instance()
-
-    //   expect(spy).toHaveBeenCalledWith(
-    //     expect.objectContaining({
-    //       action: "Impression",
-    //       entity_type: "display_ad",
-    //       unit_layout: "panel",
-    //     })
-    //   )
-    // })
+    it("tracks impressions", () => {
+      const { wrapper, instance, spy } = getTracker()
+      instance.trackImpression()
```

Comment:
I like the getTracker concept - could be useful as a system in general to avoid repeated mounting of the component that is currently happening in positron.